src/test/java/com/google/cloud/spanner/jdbc/JdbcGrpcErrorTest.java

```diff
@@ -430,7 +430,7 @@ public void transactionalCreateSession() throws SQLException {
 
   @Test
   public void readOnlyCreateSession() throws SQLException {
-    mockSpanner.setBatchCreateSessionsExecutionTime(
```

Comment:
The dialect is now also determined using the multiplexed session, meaning that `BatchCreateSessions` won't be called at all in this case.